fe/fe-core/src/main/java/com/starrocks/planner/HashJoinNode.java

```diff
@@ -68,6 +70,23 @@ public HashJoinNode(PlanNodeId id, PlanNode outer, PlanNode inner, JoinOperator
         super("HASH JOIN", id, outer, inner, joinOp, eqJoinConjuncts, otherJoinConjuncts);
     }
 
+    public boolean isSkewJoin() {
+        return isSkewJoin;
+    }
+
+    public void setSkewJoin(boolean skewJoin) {
+        isSkewJoin =
+                skewJoin && ConnectContext.get().getSessionVariable().isEnableOptimizerSkewJoinByBroadCastSkewValues();
```

Comment:
> it's better to not depend on ConnectContext here. and `ConnectContext.get().` may be null in some cases.
this is because Yangwenbo's optimization also set join's skew column, so I need to check session variables to distinguish it. This flag in HashJoinNode will be used  for runtimr filter later. After all pr merged, I wll consider to remove Wenbo's optimization, so this code line can be deleted